src/terminal-screen.c

```diff
@@ -45,6 +45,9 @@
 
 #include "eggshell.h"
 
+#define PCRE2_CODE_UNIT_WIDTH 0
```

Comment:
Hontestly, I am not sure about what this does, I got it from the patch https://sourceforge.net/p/lxde/bugs/932/attachment/0001-Don-t-use-deprecated-vte_terminal_match_add_gregex.patch

Reply:
Understtod, necessary for loading pcre2.h, otherwise it errors out.